src/Umbraco.Web.BackOffice/Controllers/UmbracoAuthorizedApiController.cs

```diff
@@ -23,6 +23,9 @@ namespace Umbraco.Cms.Web.BackOffice.Controllers
     /// is logged in using forms authentication which indicates the seconds remaining
     /// before their timeout expires.
     /// </remarks>
+    [AngularJsonOnlyConfiguration] // TODO: This could be applied with our Application Model conventions
+    [JsonExceptionFilter]
+    [AngularJsonOnlyConfiguration]
```

Comment:
@Zeegan I guess we don't need `[AngularJsonOnlyConfiguration]` twice here.

Reply:
Nice catch 💪